compiler/rustc_session/src/filesearch.rs

```diff
@@ -180,9 +180,18 @@ pub fn get_or_default_sysroot() -> Result<PathBuf, String> {
 
         // if `dir` points target's dir, move up to the sysroot
         if dir.ends_with(crate::config::host_triple()) {
-            dir.parent() // chop off `$target`
+            dir.parent() // chop off `lib` (this could be also $arch dir if the host sysroot uses a
+                // multi-arch layout like Debian or Ubuntu do)
```

Comment:
Could you re-check the suggestion please

Reply:
> Could you re-check the suggestion please

I am very sorry for the mistake. I have fixed the issue now.